doc/setup-binderhub.rst

```diff
@@ -1,40 +1,56 @@
 Set up BinderHub
 ================
 
+.. note::
+
+   Currently BinderHub requires that you configure the DNS of a web
+   address to point to the BinderHub IP address. This means that you need to
+   have control over a URL such as ``www.mydomain.com``.
```

Comment:
Specifically, it's the use of `binderhub-deploy/support` to setup letsencrypt that has this requirement. If these docs update to only cover using the binderhub helm-chart directly (resolving #114), then this requirement goes away as well.

But presumably we also want to cover how to enable LEGO as well, at least at some point.

Reply:
now that binderhub can be installed with helm, can we edit this note to say this is only necessary w/ letsencrypt?

Reply:
yes